### DISCUSSION TITLE
[Pilot] How to add new lines to "What to test" field

### DISCUSSION THREAD

**Discussion by jakunico** [fastlane/fastlane#20864] [General]
Hello folks, I'm using `\n` to enter new lines in the "What to test" message in TestFlight as follows:

```
fastlane pilot upload --changelog "${GIT_CLONE_COMMIT_MESSAGE_SUBJECT}\n${GIT_CLONE_COMMIT_MESSAGE_BODY}\n${GIT_CLONE_COMMIT_HASH}\n\Submitted by Bitrise workflow ${BITRISE_TRIGGERED_WORKFLOW_ID}"
```

Unfortunately the new lines are not processed as new lines as seen below:

![image](https://user-images.githubusercontent.com/1474405/202049390-83535ec7-e8b2-4d43-8d74-51d5c6901f33.png)

How can I achieve what I want?

Thanks!

**Comment by potej** [comment]
I have the same problem. `\\n`  won't work either.
